vcnc-server/vcnc-sampler/app.js

```diff
@@ -87,31 +84,51 @@ function requestHandler(request, response) {
   response.end();
 }
 
-const server = http.createServer(requestHandler);
-server.setTimeout(5000, () => {
-});
+// Create server
+//
+let server;
+function startServer(h, p) {
+  server = http.createServer(requestHandler);
+  server.setTimeout(5000, () => {
+  });
```

Comment:
OK good.  And about holding the channel:  vda can have thousands of channels open at once.  I don't think 2min will be a problem.